frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template-resource.tsx

```diff
@@ -74,7 +75,16 @@ export const VMTemplateDetailsList: React.FC<VMTemplateResourceListProps> = ({
           DASH
         )}
       </dd>
-      <dt>Flavor</dt>
+      <dt>
+        Flavor
+        {canUpdateTemplate && (
+          <button
```

Comment:
can we create component out of this ? Something like `EditButton` ?

Reply:
sure, done